site/static/css/docs.css

```diff
@@ -14,13 +14,14 @@ pre {
   overflow-x: auto;
   box-shadow: none;
   padding: 1rem;
+  border-radius: 5px;
 }
 
-main > h1 {
+main>h1 {
```

Comment:
Your Prettier in VSCode doesn't format CSS properly, please fix this

Reply:
I don't have prettier. I only use the `XO` plugin for now. I was contributing to one of [sindresorhus](https://github.com/sindresorhus) projects where my prettier plugin was causing some errors so had to remove it and use xo plugin. I'll have to switch back I guess then

Reply:
@talentlessguy fixed!

Reply:
@RocktimSaikia your editor must be using a prettier and eslint config defined in the project root (for the future), but it looks good so merging